R/junction-read-count.R

```diff
@@ -1,7 +1,7 @@
 #' Calculate the total number of junction reads overlapping with the introns of
 #' each promoter for the input junction file
```

Comment:
`calculateJunctionReadCounts` previously used `exonReducedRanges` as an argument.
This is replaced with `promoterCoordinates`, which contain the same fields `intronId` and `promoterId` which are used here.